### DISCUSSION TITLE
Rename `anu-vue` to `@anu-vue/core`

### DISCUSSION THREAD

**Discussion by cpreston321** [jd-solanki/anu#113] [General]
I think it would be viable to make the change earlier as it's still on going, this will make all packages follow the same monorepo namespace. I think this will make it clear since you already have `@anu-vue/nuxt` and `@anu-vue/preset-theme-default`. This will fall under all the same pattern.

**Comment by jd-solanki** [comment]
I am thinking of releasing experimental TailwindCSS support in an upcoming release. If we go with this as major release and if we all agree on renaming the package, we can surely do this.

All we have to take care of is all the existing apps will break if they upgrade without knowing that the package is renamed.

Do you know any of package that is renamed?

**Reply by cpreston321** [reply]
I've seen `nuxt` do it quite a lot, but the best way is to make an announcement on social and github talking about the rename happening within the coming weeks to then expect it coming then you use https://github.com/tiaanduplessis/pkg-rename, this will handle most of the messaging for you to tell the user to use new package name.

**Reply by jd-solanki** [reply]
https://twitter.com/me_jd_solanki/status/1609205520510115842
